src/test/java/com/kaching/logging/WithMockSecurityContextFactory.java

```diff
@@ -0,0 +1,19 @@
+package com.kaching.logging;
+
+import org.springframework.security.authentication.TestingAuthenticationToken;
+import org.springframework.security.core.context.SecurityContext;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.oauth2.jwt.Jwt;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.security.test.context.support.WithSecurityContextFactory;
+
+public class WithMockSecurityContextFactory implements WithSecurityContextFactory<WithMockUser> {
+  @Override
+  public SecurityContext createSecurityContext(WithMockUser withMockUser) {
```

Comment:
We dont need this anymore

Reply:
We will delete it when we finish implementing firebase into the project. Just leave it here for remind :>